<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneratorDialog</class>
 <widget class="QDialog" name="GeneratorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>290</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/images/image/serial.png</normaloff>:/images/image/serial.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="parametersBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>200</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Parameters</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="5" column="1">
     <widget class="QComboBox" name="flowControlBox"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="parityLabel">
      <property name="text">
       <string>Parity:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="baudRateLabel">
      <property name="text">
       <string>BaudRate:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="baudRateBox">
      <property name="toolTip">
       <string>serial port baudrate</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="flowControlLabel">
      <property name="text">
       <string>Flow control:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="stopBitsLabel">
      <property name="text">
       <string>Stop bits:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="dataBitsBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="dataBitsLabel">
      <property name="text">
       <string>Data bits:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="parityBox"/>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="stopBitsBox"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="portNameLabel">
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="serialPortInfoListBox">
      <property name="toolTip">
       <string>available serial ports</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>202</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cancelBtn">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>96</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="applyButton">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
